{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2018-17860",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https:\/\/docs.cloudera.com\/documentation\/other\/security-bulletins\/topics\/Security-Bulletin.html#concept_vp4_q2x_thb",
                    "name": "https:\/\/docs.cloudera.com\/documentation\/other\/security-bulletins\/topics\/Security-Bulletin.html#concept_vp4_q2x_thb",
                    "refsource": "CONFIRM",
                    "tags": []
                },
                {
                    "url": "https:\/\/www.cloudera.com\/documentation\/other\/security-bulletins\/topics\/Security-Bulletin.html#hadoop",
                    "name": "https:\/\/www.cloudera.com\/documentation\/other\/security-bulletins\/topics\/Security-Bulletin.html#hadoop",
                    "refsource": "CONFIRM",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "Cloudera CDH has Insecure Permissions because ALL cannot be revoked.This affects 5.x through 5.15.1 and 6.x through 6.0.1."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": [],
    "publishedDate": "2019-11-26T15:15Z",
    "lastModifiedDate": "2019-11-26T18:30Z"
}